package.json

```diff
@@ -27,7 +27,8 @@
     "url": "https://github.com/gribnoysup/react-yandex-maps.git"
   },
   "peerDependencies": {
-    "react": "^0.14.0 || ^15.0.0"
+    "react": "^0.14.0 || ^15.5.0",
```

Comment:
`prop-types` library is compatible with react `^0.14.9` or `^15.3.0`, so I think we should bump the minimal `0.14` version and lower `15` minimal version to make this library available for more people